src/ngircd/irc-server.c

```diff
@@ -320,6 +320,27 @@ IRC_NJOIN( CLIENT *Client, REQUEST *Req )
 		IRC_WriteStrChannelPrefix(Client, chan, c, false,
 					  "JOIN :%s", channame);
 
+		/* If the client is connected to me... */
```

Comment:
I'm wondering if we can refactor `IRC_JOIN()` and `join_send_topic()` to use a common function for both `JOIN` and `NJOIN` to "hello clients"? Right now this duplicates some code. Hm … 🤔